html/Corporate/style.scss

```diff
@@ -0,0 +1,664 @@
+@import url("https://fonts.googleapis.com/css?family=Exo:300,400,700&display=swap");
+
+$font-primary: "Exo", sans-serif;
+$color-primary: #3a3a3a;
+$color-secondary: #f04c4c;
+$section-bg-color: #fcfcfb;
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: $font-primary;
+  box-sizing: border-box;
+}
+
+@media screen and (min-width: 1024px) {
+  html {
+    font-size: 1.2rem;
+  }
+}
+
+body {
+  color: $color-primary;
+}
+
+h2 {
+  font-weight: lighter;
+}
+
+@mixin section-type() {
+  color: $color-secondary;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 3.2px;
+}
+
+@mixin section-title() {
+  font-size: 1.7rem;
+  letter-spacing: 1.6px;
+  font-weight: normal;
+}
+
+@mixin section-subtitle() {
+  font-size: 1.2rem;
+  font-weight: normal;
+}
+
+@mixin section-description {
+  font-size: 0.8rem;
+  line-height: 1.6;
+  opacity: 0.8;
+}
+
+@mixin vertical-padding($top: 100px, $bottom: 100px) {
+  padding-top: $top;
+  padding-bottom: $bottom;
+}
+
+@mixin default-padding ($left: 10%, $right: 10%, $top: 100px, $bottom: 100px) {
```

Comment:
in case like with default values you can use shortcut padding and don't use mixin at all